app/helpers/page_helper.rb

```diff
@@ -5,15 +5,15 @@ def flash_messages
     render(partial: 'application/flash_messages')
   end
 
-  def grouping(_data_role, options = {}, &block)
-    tag.div(**options, &block)
+  def grouping(_data_role, options = {}, &)
```

Comment:
I'm not keen on these truncations to just an ampersand! hard enough already to follow what block was referring to!